runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java

```diff
@@ -178,6 +178,7 @@
 
 	static class Minimap
 	{
+		static final int MINIMAP_WIDGET = 17;
```

Comment:
This class is supposed to be for widget group 160, and this is not a part of that

Reply:
Should I use the Viewport class instead?